__tests__/index.test.js

```diff
@@ -335,4 +335,99 @@ describe(`DELETE /api/comments/:comment_id tests`, () => {
     })
+})
+
+describe(`GET /api/articles tests V2 including queries  `, () => {
+    describe(`GET tests for sortby`, () => {
+        test(`/api/articles?sort_by=topic&order_by=asc returns correctly`, () => {
+            return request(app)
+            .get("/api/articles?sort_by=topic")
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body;
+                expect(articles)
+                .toBeSortedBy('topic', {descending: true})
+                articles.forEach(article => 
+                    expect(article).toEqual(
+                        expect.objectContaining({
+                            author: expect.any(String),
+                            title: expect.any(String),
+                            article_id: expect.any(Number),
+                            topic: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            comment_count: expect.any(String)
+                        })
+                    )
+                )
+                expect(articles.length).toBe(12)
+            })
+        })
+        test(`/api/articles?sort_by=topic&order_by=asc returns correctly`, () => {
+            return request(app)
+            .get("/api/articles?sort_by=topic&order_by=asc")
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body;
+                expect(articles)
+                .toBeSortedBy('topic', {descending: false})
+                articles.forEach(article => 
+                    expect(article).toEqual(
+                        expect.objectContaining({
+                            author: expect.any(String),
+                            title: expect.any(String),
+                            article_id: expect.any(Number),
+                            topic: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            comment_count: expect.any(String)
+                        })
+                    )
+                )
+                expect(articles.length).toBe(12)
+            })
+        })
+        test(`/api/articles?topic=mitch returns correctly`, () => {
+            return request(app)
+            .get("/api/articles?topic=mitch")
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body;
+                expect(articles)
+                .toBeSortedBy('created_at', {descending: true})
+                articles.forEach(article => 
+                    expect(article).toEqual(
+                        expect.objectContaining({
+                            author: expect.any(String),
+                            title: expect.any(String),
+                            article_id: expect.any(Number),
+                            topic: expect.any(String),
```

Comment:
This at the moment would pass even if it doesnt filter correctly. 
Could you expect 
`topic: 'mitch`

That way you would know all the objects have been filtered correctly.

Reply:
Edited topic: to `mitch` test passed